python/tchannel/tcurl.py

```diff
@@ -289,7 +289,7 @@ def main(argv=None):
     if args.verbose:
         log.setLevel(logging.DEBUG)
 
-    tchannel = TChannel()
+    tchannel = TChannel(caller_name='tcurl')
```

Comment:
to avoid confusion; it might be nice to call this `tcurl.py`. We can call the node one either `tcurl` or `tcurl.node`.